### PR TITLE
AoC 2.3 changes

### PR DESCRIPTION
**Pull request by chrisahl**
Based on reviews done as part of https://issues.redhat.com/browse/AAP-13041

### CODE REVIEW

**Comment by chrisahl** [comment]
image

**Comment by chrisahl** [comment]
image

**Review by zkayyali812 — APPROVED** [review]

File: stories/topics/proc-aws-deploy-new-vpc.adoc [hunk 1]
```diff
@@ -10,13 +10,14 @@ This procedure creates a new VPC network and deploys the application in the crea
 . In the *Software version* field, confirm the latest version available in the list is selected.
 . In the *Region* field, select the region with your EC2 key pair. 
 The CloudFormation Stack deploys in the same region. 
-. Click btn:[Continue to Launch].
-. On the following screen, click btn:[Launch]. 
+. Click btn:[Continue to Launch]. 
+. On the following screen, in the *Choose Action* field, select the launch action of *Launch CloudFormation*.
```
Comment (ianf77) [inline stories/topics/proc-aws-deploy-new-vpc.adoc:14]: , select * LaunchCloudFormation *.

Comment (chrisahl) [reply stories/topics/proc-aws-deploy-new-vpc.adoc]: Fixed

**Review by ianf77 — APPROVED** [review]
LGTM

**Pull request merged**